src/components/Restoration/SummaryRestoreBusiness.vue

```diff
@@ -0,0 +1,173 @@
+          <v-col cols="12" sm="3" class="pr-4">
+            <label>Approval Type</label>
+          </v-col>
+
+          <v-col cols="12" sm="9">
+            <template v-if="getRestoration.approvalType === ApprovalTypes.VIA_COURT_ORDER">
+              <label>Approved by Court Order</label><br/>
+              <p id="court-order-number">Court Order Number: {{ getRestoration.courtOrder.fileNumber }}</p>
+            </template>
+
+            <template v-else-if="getRestoration.approvalType === ApprovalTypes.VIA_REGISTRAR">
+              <label>Approved by Registrar</label>
+              <p id="notice-date">BC Gazette publish date: {{ noticeDate }}</p>
+              <p id="application-date">Application for Restoration mailed date: {{ applicationDate }}</p>
+            </template>
+          </v-col>
+        </v-row>
+      </article>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Watch } from 'vue-property-decorator'
+import { DateMixin } from '@/mixins'
+import { Getter, Action } from 'vuex-class'
+import { ApprovalTypes, CorpTypeCd, RouteNames, RestorationTypes } from '@/enums'
+import { NameTranslationIF, RestorationStateIF } from '@/interfaces'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+@Component({
+  mixins: [
+    DateMixin
+  ]
+})
+export default class SummaryRestoreBusiness extends Vue {
+  @Getter getEntityType!: CorpTypeCd
+  @Getter getNameRequestApprovedName!: string
+  @Getter getNameTranslations!: NameTranslationIF[]
+  @Getter getRestoration!: RestorationStateIF
+  @Getter isRestoreBusinessNameValid!: boolean
+
+  // for template
+  readonly RouteNames = RouteNames
+  readonly ApprovalTypes = ApprovalTypes
+
+  /** The entity description. */
+  get entityDescription (): string {
+    return GetCorpFullDescription(this.getEntityType)
+  }
+
+  get isLimitedRestoration (): boolean {
+    return (this.getRestoration.type === RestorationTypes.LIMITED)
+  }
+
+  get expiry (): string {
+    return this.yyyyMmDdToPacificDate(this.getRestoration.expiry)
+  }
+
+  get relationships (): string {
+    return this.getRestoration.relationships.join(', ')
+  }
+
+  get noticeDate (): string {
+    return this.yyyyMmDdToPacificDate(this.getRestoration.noticeDate)
+  }
+
+  get applicationDate (): string {
+    return this.yyyyMmDdToPacificDate(this.getRestoration.applicationDate)
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.restoreBusinessStepErrorMessage {
+  padding-top: 1.25rem;
+  padding-left: 1.25rem;
+  color: $app-red;
+}
+
+.v-icon.mdi-information-outline {
+  margin-top: -2px;
+}
+
+#company-name {
+  font-size: $px-22;
+  font-weight: bold;
+  color: $gray9;
+}
+
+p {
+  margin-bottom: 0;
+}
+
+#company-description,
+.name-translation,
+#expires-on,
+#applicants-relationships,
+#court-order-number,
+#notice-date,
+#application-date {
+  color: $gray7;
```

Comment:
I would have just used a class, but we already declared the IDs for unit testing.